packages/css/src/components/grid/grid.scss

```diff
@@ -9,8 +9,8 @@
   --amsterdam-grid-gap: var(--amsterdam-grid-spacious-gap);
   --amsterdam-grid-padding-inline: var(--amsterdam-grid-spacious-padding-inline);
 
+  column-gap: var(--amsterdam-grid-gap);
```

Comment:
This removes the row gap by default, which is not what we want. We rely on the vertical gap being the same as the horizontal one by default, e.g. when moving cells around respponsively and in grids of cards or top task buttons.